docs/intro.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to the High-Speed Data Transmission Customer Portal
 
-High-speed Data Transmission (HDT) is designed to optimize TCP/UDP application acceleration with superior performance and benefits. The HDT Customer Portal is a web-based user interface for the HDT platform, where you can add, view, and manage your transports and retrieve detailed reports about the usage.
+The High-Speed Data Transmission (HDT) Customer Portal is a web-based user interface for the HDT platform, where you can add, view, and manage your transports and retrieve detailed reports about the usage.
```

Comment:
Because "High Speed Data Transmission" is a proper name, there is no need for the hyphen between "High" and "Speed." (source: https://www.chicagomanualofstyle.org/qanda/data/faq/topics/HyphensEnDashesEmDashes/faq0112.html#:~:text=First%2C%20capitalized%20proper%20nouns%20are,%2D)%20to%20an%20open%20compound.)

Reply:
Hi, Marc, I think it is for the acronym "HDT", the "S" need to be lowercase, then the "High speed Data Transmission" feels a little bit strange, I searched around, found both "high speed" and "high-speed" are widely used, maybe we can keep this "High-speed Data Transmission (HDT)"?